### PR TITLE
Feature/color theme

### PR DESCRIPTION
**Pull request by roblack**
## Checklist
_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](https://github.com/RocketFanOrg/RocketFanApp/master/CONTRIBUTING.md) doc
- [x] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] My changes do not generate new SwiftLint warnings
- [ ] Add GitHub issue #

## Types of changes

What types of changes does your code introduce to RocketFanApp?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Work done
Give a brief summarized description of what the PR is for and the work done in it.
Defined colors in a separate Asset Catalogue and added a Theme enum in "Utilities" folder.

## Notes
Any additional notes will come here, use as below if no notes are required.
Not sure if Theme should have been placed in Styles folder though. Figured its a singular enum but really 50/50 on this one.

## Screenshots
Please add screenshots of the different states found in this PR.

### CODE REVIEW

**Comment by roblack** [comment]
@squarefrog @RJ-Clegg 
forgot to @ you :)

File: RocketFan/Utilities/Theme.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
+//  Theme.swift
+//  RocketFan
+//
+//  Created by Emin Roblack on 17/06/2020.
+//  Copyright © 2020 RocketFanOrg. All rights reserved.
+//
+
```
Comment (squarefrog) [inline RocketFan/Utilities/Theme.swift:8]: Don't need any of these header comments

File: RocketFan/Utilities/Theme.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import SwiftUI
+
+enum Theme {
+    static var background:     Color {     Color("background") }
```
Comment (squarefrog) [inline RocketFan/Utilities/Theme.swift:12]: I'd generally prefer this without white spacing:

```
static var background: Color { Color("background") }
```

Comment (roblack) [reply RocketFan/Utilities/Theme.swift:12]: Will do! Thought this might be "frowned" upon 👍

File: RocketFan/Utilities/Theme.swift (hunk 1 above)
Comment (roblack) [reply RocketFan/Utilities/Theme.swift:8]: 👍 I wish i could actually remove this by default.

**Comment by squarefrog** [comment]
If you delete the empty `Icons.xcassets` catalogue, bitrise should build successfully.

**Comment by roblack** [comment]
Done, waiting for the check to go through.

**Pull request closed without merging**